WorkloadManagementSystem/Service/JobStateUpdateHandler.py

```diff
@@ -57,7 +60,7 @@ def initialize(self):
     return S_OK()
 
   ###########################################################################
-  types_updateJobFromStager = [[basestring, int, long], basestring]
+  types_updateJobFromStager = [[six.string_types, int, int], six.string_types]
```

Comment:
[six.string_types, int, int] <- why int, int ? typo ? should be kept int, long, I guess

Reply:
There is no `long` type in python 3

Reply:
Then it should be just [six.string_types, int]

Reply:
@atsareg Actually these changes are automated by the `python-modernize` tool. Some of these changes need to be discussed due to such issues.

Reply:
No need to repeat `int, int`: just once is enough. This is valid here and in the following lines.